Go/Bitcoin_Price_Tracker/Readme.md

```diff
@@ -0,0 +1,24 @@
+# Bitcoin_Price_Tracker
+
+A script used to fetch the live price of 1 BTC in INR, written in GOlang. [Colly package](https://github.com/gocolly/colly) is used as a base for the web scraping and tag search.
+
+- Fetches the live price of 1 Bitcoin.
+
+## Setup instructions
+
+After forking the repository and cloning on user system ,follow these commands on git bash/bash:
```

Comment:
Keep it `After forking the repository on your local machine, follow these commands on your terminal:`